templates/control/2fa/disable.html

```diff
@@ -0,0 +1,29 @@
+$def with (username, error)
+$:{RENDER("common/stage_title.html", ["Disable 2FA", "Two-Factor Authentication", "/control/2fa/status"])}
+
+<div class="content form clear">
+  <form method="POST" class="form skinny clear" action="/control/2fa/disable">
+    $if error == "2fa":
+      <div id="signin_error"><strong>Whoops!</strong> The 2FA response token or recovery code you provided did not validate.</div>
+    $elif error == "verify":
+      <div id="signin_error"><strong>Hey!</strong> Did you want to disable 2FA? You didn't check the verification checkbox!</div>
+    <h3>Disable Two-Factor Authentication (2FA)</h3>
+    <p><strong>Username: $:{MARKDOWN_EXCERPT(username)}</strong></p>
```

Comment:
Oh! Got it. I was unaware that was the difference between ``${foo}`` and ``$:{foo}``. Noted for the future. Apologies if there seemed to be any confusion there.